plugins/modules/aci_rest.py

```diff
@@ -379,7 +379,6 @@ def main():
         with open(src, "r") as config_object:
             # TODO: Would be nice to template this, requires action-plugin
             payload = config_object.read()
-            payload_output_file = json.loads(payload)
```

Comment:
Can you run a test that specifies output_path as input variable? This might cause issues for code part near the end.

Reply:
Yes. We need to check that.